docs/rbac.md

```diff
@@ -101,3 +101,58 @@ kind: ConfigMap
 metadata:
   name: argocd-rbac-cm
 ```
+## Project Roles
```

Comment:
@dthomson25  - I was not aware of projects (since I used this https://github.com/argoproj/argo-cd/blob/master/docs/getting_started.md ). If customers already know about projects, its okay. But if they do not, can we add a link to docs?

Reply:
After talking to @sindhusrinivasan27, we both agreed that projects are defined earlier in the RBAC page.  This makes it reasonable to assume that the customer will have knowledge of projects when they reach this section of the documentation.